src/main/java/Duke.java

```diff
@@ -1,10 +1,138 @@
+import duke.tasks.Task;
+import duke.tasks.Deadline;
+import duke.tasks.Event;
+import duke.tasks.Todo;
+import duke.exceptions.EmptyDescriptionException;
+import duke.exceptions.TaskToMarkDoesNotExistException;
+import duke.exceptions.UnknownCommandException;
+
+import java.util.Scanner;
+
 public class Duke {
+    private static void displayList(Task[] tasks) {
+        System.out.println("Here are the tasks in your list:");
+        for (int i = 0; i < Task.numberOfTasks; i += 1) {
+            System.out.print(i + 1 + ". ");
+            tasks[i].printTask();
```

Comment:
This is a good usage of polymorphism. When you are implementing the features in future levels, you might want to think about using similar techniques for other appropriate features too.